amplipi/ctrl.py

```diff
@@ -236,6 +236,17 @@ def reinit(self, settings: models.AppSettings = models.AppSettings(), change_not
     for major, minor, ghash, dirty in self._rt.read_versions():
       fw_info = models.FirmwareInfo(version=f'{major}.{minor}', git_hash=f'{ghash:x}', git_dirty=dirty)
       self.status.info.fw.append(fw_info)
+    if self.status.info.fw:
+      fw_info = self.status.info.fw[0]
+      logger.info(f"Main unit firmware version: {fw_info.version}")
+    else:
+      logger.info(f"No preamp")
+    if self.status.info.fw[1:]:
+      for i in range(1,len(self.status.info.fw)):
+          fw_info = self.status.info.fw[i]
+          logger.info(f"Expansion unit {i} firmware version: {fw_info.version}")
+    else:
```

Comment:
A potentially cleaner version
```suggestion
    for i, fw_info in enumerate(self.status.info.fw[1:], start=1):
      logger.info(f"Expansion unit {i} firmware version: {fw_info..version}")
    else:
```